django_slack_oauth/pipelines.py

```diff
@@ -0,0 +1,14 @@
+from django_slack_oauth.models import SlackOAuthRequest
+
+
+def log_request(request, api_data):
+        SlackOAuthRequest.objects.create(
+            access_token=api_data.pop('access_token'),
+            extras=api_data
```

Comment:
@izdi On second thought, getting the IP is pretty opinionated and depends on the deployment/setup. For example, do we get it from `HTTP_X_FORWARDED_FOR` or from `REMOTE_ADDR`? If we're hosting behind an ELB, then we can trust the `HTTP_X_FORWARDED_FOR` but only if we've set up NGINX correctly. With some bad intentions, a user could fake it. See: http://stackoverflow.com/questions/18264304/get-clients-real-ip-address-on-heroku#answer-18517550